about.html

```diff
@@ -0,0 +1,193 @@
+<html>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<head>
+  <title>À propos</title>
+  <link rel="stylesheet" href="css/bootstrap.min.css">
+  <link rel="stylesheet" href="/css/style.css">
```

Comment:
Why the first import starts with `css/...` and the second one with `/`, like `/css/...`